core/include/vecmem/containers/impl/jagged_vector_buffer.ipp

```diff
@@ -14,6 +14,8 @@
 #include <numeric>
 #include <vector>
 
+#include "../details/aligned_multiple_placement.hpp"
```

Comment:
Just because we didn't use relative path names in other places either, I would go with `vecmem/containers/details/aligned_multiple_placement.hpp` here as well. But of course this is a largely subjective point.